src/modlunit/declare.cpp

```diff
@@ -7,7 +7,6 @@
 #include <strings.h>
```

Comment:
keeping this conversation open until we check nrn-modeldb-ci

Reply:
I see the word, "Success" in https://github.com/neuronsimulator/nrn-modeldb-ci/actions/runs/3181158338
That sounds like we are ready to merge.

Reply:
it means that reports have been successfully generated, still need to personally validate them

Reply:
All OK